docs/modules/setup/pages/standalone-executable.adoc

```diff
@@ -52,3 +52,4 @@ You can open this file in a browser.
 
 TIP: Alternatively you can use `:revealjsdir: https://cdn.jsdelivr.net/npm/reveal.js` to avoid having to setup reveal.js locally.
 Just be aware that then if you have no access to internet then the presentation will not load properly.
+Also, when using the CDN, the speacker notes will be send as plain text - instead of HTML - due to security reasons.
```

Comment:
```suggestion
Also, when using jsDelivr, speaker notes will be served as plain text - instead of HTML - for https://www.jsdelivr.com/documentation#id-be-advised-that-you-must-include-file-extension-in-the-values-for-example-main-index--this-will-not-work-main-indexjs--this-is-the-correct-way-restrictions[security reasons] and thus will not work.
```